frontend/cypress/integration/featureFiles/masthead.feature

```diff
@@ -0,0 +1,38 @@
+@masthead
+@multi-cluster
+# don't change first line of this file - the tag is used for the test scripts to identify the test suite
+
+Feature: Kiali masthead cluster components status
+
+  User wants to verify the Kiali masthead content
+
+  Background:
+    Given user is at administrator perspective
+    And user is at the "overview" page
+
+  @multi-cluster
+  Scenario: Cluster components healthy
+    Then user sees "east" cluster label with a "success" icon
+    When user hovers over the cluster icon
+    Then user sees a tooltip with text "east"
+    Then user sees a tooltip with text "west"
+    Then user does not see any "Not" in the tooltip
+    Then user does not see any "Unreachable" in the tooltip
+    Then user hovers out the cluster icon
```

Comment:
What is this last assertion testing? Seems like it performs an action but does no assertions afterward?

Reply:
This was like a cleanup, to close the tooltip, will move it to separate after

Reply:
removed